content/reference-transaction-format.md

```diff
@@ -25,6 +26,63 @@ Any signature type can authorize any type of transaction, with the following exc
 * You can never remove the last method of signing transactions from an address.
 
+
+## Understanding Master and Regular Keys
+
+In the XRP Ledger, a digital signature proves that a transaction is authorized to do a specific set of actions. A digital signature is created based on a key pair associated with the transaction's sending account. There are two types of key pairs available: master keys and regular keys. Each key pair includes a private key and a public key.
+
+***TODO: I feel like some concept docs on private and public keys would be great -- like when to use them. Talk about the master_key, master_seed, master_seed_hex, public_key, and public_key_hex, for example, and why we provide them and what are the use cases. It would also be good to provide concept doc on how keys are used to generate the "signature" and the different types of signature-like values that we generate: Signature, SigningPubKey, and TxnSignature, for example. What are the use cases? I believe this info is in the doc, but discussed in the context of tutorials.***
+
+### Master Keys
+
+When you create an account using the [`wallet_propose`](reference-rippled.html#wallet-propose) method, the `master_seed` (private key) and `public_key` returned in the response are the **master keys** for the account. These master keys are an intrinsic part of the account's identity as the `account_id` (account address), also returned in the `wallet_propose` response, and the `public_key` value are mathematically related.
+
+Here's a sample `wallet_propose` response, including the `master_seed` (private key), `public_key`, and `account_id` (account address) values:
+
+```
+{
+  "result": {
+    "account_id": "rM8oJwkKtgEU2HZkCHsJScLHqegWpgBRxt",
+    "key_type": "secp256k1",
+    "master_key": "SAY ARMY KITE LETS PAN WAVE JUKE ACRE MAT TILT FEAT LED",
+    "master_seed": "PrivateKeyXXXXXXXXXXXXXXXXXXX",
+    "master_seed_hex": "47E8F7F59D0CC9A3F01FD3D69CCAE93A",
+    "public_key": "aBQEK3r6hb8isnCWmortkNdv3MSMKP74o7gj3kHZQF9vDkbXo4Qs",
+    "public_key_hex": "032D911AE9887278110A8FE25169546D4E95A82AF5EFA25525FC3FDFFDC5A8B604"
+  },
+  "status": "success",
+  "type": "response"
+}
+```
+
+Master keys have more powerful signing authority than regular keys. In addition to being able to sign all transactions that regular keys can, master keys are the only keys that can be used to perform the following actions:
+
+* Use the master private key to [disable the master public key](#accountset-flags).
+* Permanently give up the ability to [freeze counterparties](concept-freeze.html#no-freeze).
+
+**Important:** Master keys can be disabled, but cannot be changed. This means that if your master private key is [compromised](concept-issuing-and-operational-addresses.html), you cannot change it to stop the compromise.
+
+This is a compelling reason to keep your master keys offline whenever possible and set up regular keys to sign transactions from your master account instead.
```

Comment:
There's no such thing as a "master account" in this context. Every account is just an account.

Reply:
I totally made up that construct 💯  ... I think as a bridge to help me get to the next level of understanding. Removed it now that your help in refining the other content has bridged the gap.